crates/proof-of-sql/src/sql/proof_gadgets/range_check_test.rs

```diff
@@ -27,6 +27,7 @@ impl ProverEvaluate for RangeCheckTestPlan {
         table_map: &IndexMap<TableRef, Table<'a, S>>,
     ) -> Table<'a, S> {
         builder.request_post_result_challenges(1);
+        builder.produce_one_evaluation_length(256);
```

Comment:
@Dustin-Ray Is this really just so that we update `range_length`? Looks like this is the right time to split the one evaluation length queue from the `range_length` updates. Do you want to do this or do you want me to?

Reply:
@iajoiner if its a small change without much of a diff then I would be happy to do it in this PR, if its more involved I can definitely do it in another PR